app/E-Commerce/payment.java

```diff
@@ -0,0 +1,70 @@
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import com.stripe.android.PaymentConfiguration;
+import com.stripe.android.PaymentSession;
+import com.stripe.android.PaymentSessionConfig;
+import com.stripe.android.view.PaymentSheet;
+import com.stripe.android.view.PaymentSheetResult;
+import com.stripe.android.view.PaymentSheetResultCallback;
+import com.ispc.gymapp.R;
+
+public class PaymentMethod extends AppCompatActivity {
+
+    private PaymentSheet paymentSheet;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        PaymentConfiguration.init(getApplicationContext(), "tu_clave_secreta_de_stripe");
```

Comment:
PaymentConfiguration.init(getApplicationContext(), "tu_clave_secreta_de_stripe");
```suggestion
        PaymentConfiguration.init(getApplicationContext(), );
        
```
cual es la clave?